native/effektio/src/api/receipt.rs

```diff
@@ -111,10 +111,7 @@ impl ReceiptController {
         let me = self.clone();
         client.add_event_handler_context(me);
         let handle = client.add_event_handler(
-            |ev: SyncEphemeralRoomEvent<ReceiptEventContent>,
-             room: MatrixRoom,
-             Ctx(me): Ctx<ReceiptController>,
-             handle: EventHandlerHandle| async move {
+            |ev: SyncReceiptEvent, room: MatrixRoom, Ctx(me): Ctx<ReceiptController>| async move {
```

Comment:
same question about `Ctx(me)` here.

Reply:
Please read this code snippet: https://github.com/matrix-org/matrix-rust-sdk/blob/f8b02d1a11d07d28ce87132b93d30d6b03d76b5a/crates/matrix-sdk/src/event_handler/mod.rs#L422